<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>export_lyrics</class>
 <widget class="QDialog" name="export_lyrics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导出歌词</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="4">
    <widget class="QLineEdit" name="save_path_lineEdit"/>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QComboBox" name="lyricsformat_comboBox">
     <item>
      <property name="text">
       <string>LRC(逐字)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LRC(逐行)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>增强型LRC(ESLyric)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SRT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ASS</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>歌词格式:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="translate_checkBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>译文</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QComboBox" name="filename_mode_comboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>133</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>依照格式(根据歌词中的信息)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>依照格式(根据歌曲中的信息)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>与歌曲文件名相同</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="save_path_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>选择保存路径</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>歌词文件名:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>歌词类型:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="original_checkBox">
     <property name="text">
      <string>原文</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QCheckBox" name="romanized_checkBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>罗马音</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>export_lyrics</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>export_lyrics</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
